### DISCUSSION TITLE
autocomplete not showing my variables?

### DISCUSSION THREAD

**Discussion by gstlouisgit** [atom/atom#23631] [General]
I was using sublime and when typing i would see my declared variable in that page.  Does Atom have this?  I've installed autocomplete-plus, and I still don't have it.  I thought I saw it the other day, but it doesn't look like I can see any of them

**Comment by Royassink** [comment]
because you me i twist you ton me certificate

Op vr 18 feb. 2022 21:04 schreef gstlouis ***@***.***>:

**Reply by shaggytofly** [reply]
> ] What_is_atom~~use~~for~~~
